.github/workflows/cibuildwheel.yaml

```diff
@@ -153,6 +153,15 @@ jobs:
           - cp312
           - cp313
           - cp314
+        include:
+          - platform:
+              name: manylinux_x86_64
```

Comment:
no arm?

Reply:
No, I was thinking we could start small with just x86. Other platforms can still use the pure Python wheel. Or do you think we should add ARM too?

Reply:
No, there shouldn't be any except more wheels to produce, and more storage used on PyPI.